### DISCUSSION TITLE
WSA 2310 Update

### DISCUSSION THREAD

**Discussion by jaholme** [microsoft/WSA#466] [Announcements]
# What's New

We've shipped an update for Windows Subsystem for Android™ on Windows 11 to the [Windows Subsystem for Android™ Preview Program](https://learn.microsoft.com/en-us/windows/android/wsa/preview-program). This update (2310.40000.0.0) includes improvements to platform reliability and functionality improvements.

## Release Notes

- Platform reliability improvements
- Gamepad triggers are no longer reversed, a fix for #452 

# Giving feedback

If you are having issues with Windows Subsystem for Android™ – please file feedback via Feedback Hub under Apps > Windows Subsystem for Android™. For more information about troubleshooting and submitting feedback: [Troubleshooting and FAQ for mobile apps on Windows](https://support.microsoft.com/en-us/windows/troubleshooting-and-faq-for-mobile-apps-on-windows-d6062afd-98a1-4018-a7c8-6b3b680a2ea5).

Thanks,
Windows Subsystem for Android™ Team

**Comment by vanVaals** [comment]
Warum ist Android nur mit Amazon verbunden? ich brauchte einiges aus dem Googlestore. Nein, keine Spiele

**Reply by Mitia99** [reply]
Use [this](https://github.com/MustardChef/WSABuilds) if you need WSA with Googlestore

**Reply by HotCakeX** [reply]
3rd party pre-built binaries aren't trustworthy. They can contain malware.

Run it on a system that's isolated from your personal information and files.

**Reply by Mitia99** [reply]
> 3rd party pre-built binaries aren't trustworthy. They can contain malware.
> 
> Run it on a system that's isolated from your personal information and files.

Indeed but unfortunately it's the only to use Google Playstore which is way better than Amazon appstore.

**Reply by HotCakeX** [reply]
Use aurora store
https://auroraoss.com/

I never use Amazon appstore.

**Comment by HotCakeX** [comment]
Thanks for the update, hopefully the next version will bring Android 14 to WSA.

**Reply by zoey29o** [reply]
google itself didnt launch a x86_64 Desktop Image Get Android studio with android emulators A14 exist for phones and tablets. 
of course, there is also that Goole Android emulator uses Qemu system since they include also virtualization of hardware like sensors etc even call function is supported the downside it can't start automatically at boot so for daily driving. 

if you have a samsung phone install dex for windows and then you can have full android without impacting pc performance of phone link yet again samsung phones are the most supported here since they preload link to windows

**Comment by Almighty188** [comment]
Have a question, is there a way to do screenshots directly inside WSA since some Android apps needs the images on the phone itself and I don't see a way to emulate power button + volume down.

**Reply by Mitia99** [reply]
Using **Share user folders** Advanced settings should display your screenshot taken with Windows snipping tool in your Android app as well

**Reply by Almighty188** [reply]
It doesn't as I am using apps and basically it is blank as I think it needs to find it in the DCIM/Camera folder.

**Reply by zoey29o** [reply]
well you can screenshot with sniping tool WSA windows see attached if it doesn't you might have a wrong configuration.  you need to set as independent windows, so the image won't be translated. 
![Screenshot 2023-12-23 043912](https://github.com/microsoft/WSA/assets/91636547/c707735f-ad08-4284-a1b0-6e16c7909510)
![Screenshot 2023-12-23 044040](https://github.com/microsoft/WSA/assets/91636547/e641debb-927a-44fd-a9af-ca5b8a5c1c53)
![Screenshot 2023-12-23 044123](https://github.com/microsoft/WSA/assets/91636547/5193e036-6673-4e12-9f4f-2f6d6ede2533)
![Screenshot 2023-12-23 044329](https://github.com/microsoft/WSA/assets/91636547/d0292ffb-d368-4578-a0c8-b8a93a44d659)
![Screenshot 2023-12-23 044430](https://github.com/microsoft/WSA/assets/91636547/e8592708-caea-4cf3-990e-b1a31971bb6d)
![Screenshot 2023-12-23 044525](https://github.com/microsoft/WSA/assets/91636547/b969b5f9-028a-417e-a7e7-45984d45be3a)
